apps/docs/docs/templates/ui-components.md

```diff
@@ -18,25 +18,39 @@ Example: The _add component name_ component is _add component meaning_ used to _
 <div>List component variants/types here if applicable</div>
 ```
 
+---
+
 ## Import
 
+Import the component from <a href="https://www.faststore.dev/reference/ui/get-started-faststore-ui">@faststore/ui</a>
+
 ```tsx
 import { ComponentName } from '@faststore/ui'
 ```
 
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/{folder}/{ComponentName}/styles.scss'
```

Comment:
Shouldn't the dev be importing all components from `fs/core`?

Reply:
not necessarily, they can chose to import the component style independently. (But in our starter we will import everything at once using the `@faststore/ui/src/styles/global.scss`)

Reply:
or we can be more explicit here
Import Styles (if adding the component independently)

btw, this might be a problem, for instance,  if you wish to use a component that share styles from button.scss, you also need to import its style. 🤔